wizard.sh

```diff
@@ -378,7 +378,7 @@ delete_bins() {
 # uninstall crowdsec and cscli
 uninstall_crowdsec() {
     systemctl stop crowdsec.service
-    ${CSCLI_BIN} dashboard remove -f -y || echo "metabase not running"
+    ${CSCLI_BIN} dashboard remove -f -y 2> /dev/null || echo "metabase not running"
```

Comment:
I think we might instead want dashboard remove not to `fatalf` if the dashoard isn't present